src/components/DpLabel/DpLabel.vue

```diff
@@ -49,6 +49,12 @@ export default {
       required: true
     },
 
+    hidden: {
```

Comment:
Also if this is (rather) obvious: a one-line documentation on the prop will later ease storybook completion.

Reply:
If we use class `sr-only` should we call the prop also sr-only? :thinking:

Reply:
idk.. i think while `srOnly` or `screenReaderOnly` will better describe the intention of this prop, we may want to strive for consistency here... in the button component the prop reads `hideText`... so as we're hiding the whole label here including hint and tooltip, we may go with just `hide`? idk.

Reply:
Done in [023db8c](https://github.com/demos-europe/demosplan-ui/pull/894/commits/023db8c116cc3f07c4b1e7f3f9502bc427b8be53)